sample-code/SimpleTypes.fs

```diff
@@ -766,7 +766,7 @@ type FooWithSpaceAfterNew =
 // symbol `:` and casting operator `:>` should use the symbol color.
 
 type GenericType1<'a> =
-    class end
+    abstract member Foo : 'a -> 'a
```

Comment:
> Maybe we should add an example instead of changing one?

There are already some other examples where this shows up in isolation. My rationale for putting this here is so that it can be seen alongside other uses of the colon symbol to easily verify consistency. This PR is really meant to be an addendum to #190, where I addressed inconsistencies with colons and angle brackets. This modifies one of the examples I added in #190. If you prefer, I can revert this and add something like the following:

    type AbstractType =
        abstract member Foo : unit -> unit

Should I do it this way instead?

Reply:
@TheAngryByrd I added a fixup commit. Let me know what you think.